docker-compose.yml

```diff
@@ -13,9 +13,6 @@ services:
       DBHOST: influx
       DBUSER: bwm
       DBPASSWORD: bwm
-    volumes:
```

Comment:
Why are you not creating local volumes instead of docker volumes for better portability?